src/game.cpp

```diff
@@ -9792,12 +9804,15 @@ bool game::handle_liquid( item &liquid, item * const source, const int radius,
         menu.text = string_format( _( "What to do with the %s from %s?" ), liquid_name.c_str(), m.name( *source_pos ).c_str() );
     } else if( source_veh != nullptr ) {
         menu.text = string_format( _( "What to do with the %s from the %s?" ), liquid_name.c_str(), source_veh->name.c_str() );
-    } else {
+    } else if( source_mon != nullptr ) {
+	    menu.text = string_format( _( "What to do with the %s from the %s?" ), liquid_name.c_str(),
+	                               source_mon->disp_name().c_str() );
+	} else {
```

Comment:
Tab, check all the changed files if you don't have any other missed tabs. Most text editors should support regular search detecting tabs.